RecoLocalCalo/HcalRecAlgos/plugins/HcalChannelPropertiesEP.cc

```diff
@@ -118,18 +124,28 @@ class HcalChannelPropertiesEP : public edm::ESProducer {
     return prod;
   }
 
+  ReturnType3 produce3(const HcalPFCutsRcd& rcd) {
+    const HcalTopology& htopo = rcd.getRecord<HcalRecNumberingRecord>().get(topoToken3_);
```

Comment:
This could be just
```suggestion
    const HcalTopology& htopo = rcd.get(topoToken3_);
```

Reply:
OK, thanks. I was just following the standard old recipe which requires look up of dependent records. I presume, @swagata87 can make the requested change.